content/docs/Getting started/deploying-cuebot.md

```diff
@@ -84,6 +84,20 @@ Make sure you also complete the following steps:
     to the database. This configuration is outside of the scope of this
     guide.{{% /alert %}}
 
+1.  Cuebot needs to know the location of the filesystem where render assets
+    are stored and log files are written. In a large-scale deployment a shared
+    filesystem such as NFS is often used for this purpose. For the purposes of
+    this tutorial, you can use any directory you have write access to.
```

Comment:
Nit: as tutorials are a specific type of doc, I recommend rewording to:

For the purposes of this guide, you can...

Reply:
Done.